jest.config.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  testEnvironment: 'node'
```

Comment:
``` export default { testEnvironment: 'node' }  ```

Reply:
I like this better, because is standard among all the *.config.js that exists